pkg/cli/admin/release/info.go

```diff
@@ -1564,6 +1572,13 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 		}
 	}
 
+	if skipBugCheck {
```

Comment:
I'd rather have the check inside `if !skipBugCheck` and leave the printing as is defined below. This introduces unnecessary code duplication .

Reply:
Updated